### ISSUE TITLE
KDE-store install error

### ISSUE THREAD

**Issue by tarun-varier** [vinceliuice/WhiteSur-kde#12]
I am getting this long ass error while installing from the kde store from the settings. I am actually a noob. Plz help me.
`Installation of /tmp/DlWHAo-com.github.vinceliuice.WhiteSur-dark.tar.xz failed: Could not install dependency: 'kns://sddmtheme.knsrc/api.kde-look.org/1398839'` --- I can only copy this much, but it has some special character and stuff. Again, I am a noob. Here's a screenshot of the screen I received
![2020-07-16-221222_352x541_scrot](https://user-images.githubusercontent.com/63459975/87698870-b4856000-c7b1-11ea-88b3-2bb9d308acc9.png)

Thanks a lot

**Comment by Yupro838** [comment]
Getting the same error. Did you manage to fix it?

**Comment by tarun-varier** [comment]
nope,i didn't....I just gave up (I was a noob back then)... I don't use KDE anymore, but I suppose there might be some manual way to install it and then maybe figure more stuff out... I'll check it out again.... which distro u using btw? I think I got this error from the base arch or manjaro (I distro-hop b/w these two)... thanks
